### PR TITLE
bcc growth@ on downgrade emails

### PR DESCRIPTION
**Pull request by nickpell**
https://manage.dimagi.com/default.asp?264241

should merge with https://github.com/dimagi/commcarehq-ansible/pull/1079

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -490,6 +490,7 @@
 INTERNAL_SUBSCRIPTION_CHANGE_EMAIL = 'accounts+subchange+internal@example.com'
 BILLING_EMAIL = 'billing-comm@example.com'
 INVOICING_CONTACT_EMAIL = 'billing-support@example.com'
+GROWTH_EMAIL = 'growth@example.com'
```
Comment (nickpell) [inline settings.py]: @orangejenny re: https://github.com/dimagi/commcare-hq/pull/18649#issuecomment-347277850

I'm adding the growth email here

Comment (orangejenny) [reply settings.py]: great

**Review by orangejenny — APPROVED** [review]

**Review by kaapstorm — APPROVED** [review]

File: settings.py (hunk 1 above)
Comment (emord) [reply settings.py]: Any reason these aren't empty strings? (not asking you to fix all of these, just thinking about this one for now)

If the default is `''` then we can either check to see if django's email engine will just ignore it, or only bcc if it exists. Then we don't have to worry about setting this for enterprise environments that "growth" (or these other emails) will never care about

Comment (nickpell) [reply settings.py]: @emord Seems like a reasonable approach to me

**Pull request merged**